proxy/api/src/lib.rs

```diff
@@ -22,7 +22,8 @@
     clippy::or_fun_call,
     clippy::shadow_reuse,
     clippy::clippy::option_if_let_else,
```

Comment:
```suggestion
    clippy::option_if_let_else,
```

Seems to be bogus otherwise.